### PR TITLE
Merge feature-business-registry-dashboard-updates into main

### PR DESCRIPTION
**Pull request by seeker25**
This is merging of the feature branch for various business dashboard changes into main.

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2525-0vvrop22.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2525-0vvrop22.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2525-0vvrop22.web.app

**Comment by seeker25** [comment]
Fixing a few of the sonarclouds now.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2525-0vvrop22.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2525-0vvrop22.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2525-0vvrop22.web.app

File: auth-web/src/components/auth/manage-business/manage-business-dialog/ManageBusinessDialog.vue [hunk 1]
```diff
@@ -0,0 +1,971 @@
...
+    },
+    businessLegalType: {
+      type: String,
+      default: ''
+    },
+    showBusinessDialog: {
+      type: Boolean,
+      default: false
+    },
+    isStaffOrSbcStaff: {
```
Comment (seeker25) [inline auth-web/src/components/auth/manage-business/manage-business-dialog/ManageBusinessDialog.vue:340]: This should eventually be moved to the store

**Review by Jxio — APPROVED** [review]
LGTM!

**Pull request merged**